src/components/Navbar/Navbar.styled.tsx

```diff
@@ -1,14 +1,75 @@
 import styled from 'styled-components';
+import { HiOutlineHome } from 'react-icons/hi';
+import {
+  BsReverseLayoutTextWindowReverse,
+  BsBoxArrowInDown,
+} from 'react-icons/bs';
+import { RiLogoutCircleLine } from 'react-icons/ri';
 
 // should be styled and updated later
 export const NavbarContainer = styled.div`
-  background-color: black;
+  background-color: ${({ theme }) => theme.background.primary};
+  width: 250px;
+  height: 100%;
   color: white;
-  border: 2px solid green;
-  padding: 2.5em;
+  padding: 5px;
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-around;
+`;
+export const NameLogo = styled.p`
+  font-weight: bold;
+  font-size: 20px;
+  text-align: center;
+  width: 100%;
+  margin-bottom: 2px;
 `;
-
 export const MenuItems = styled.section`
+  display: flex;
+  gap: 30px;
+  flex-direction: column;
+  justify-content: space-between;
+  color: white;
+  padding: 2.5em;
+  display: inline-flex;
+  align-items: flex-start;
+`;
+export const UserInfoContainer = styled.section`
   display: flex;
   flex-direction: column;
+  align-items: center;
+  text-align: center;
+  margin: 0 auto;
+`;
+
+export const Avatar = styled.img`
+  height: 70%;
+  border: 1px solid rgba(255, 255, 255, 0.3);
+  border-radius: 50%;
+  padding: 6px;
+  margin-bottom: 1em;
+`;
+
+export const UserData = styled.p`
```

Comment:
```suggestion
export const StyledUserInfo = styled.div`
```
if this is a styled `p`, then you should get rid of the `p` inside the `UserData` in the component. I see it's currently:
```
<UserData> <p>Name</p></UserData>
```
this means there's a nested `p` that's unnecessary

Reply:
I think I did this correctly! Let me know if I need to change anything.

Reply:
also the reason I recommend the name `StyledUserInfo` instead of `UserData`, is because `data` is usually used for the naming the variables for the actual data.